### PR TITLE
Edit Delete User

### PR DESCRIPTION
**Pull request by SSalekin**

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -13,6 +17,14 @@
       <label><%= t "email" %></label>
       <p class="form-control profile_align"><%= @user.email %></p>
     </div>
-    <div class="panel-footer panel-course-footer"></div>
+    <div class="panel-footer panel-course-footer">
+      <% if @user == current_user %>
+        <%= link_to t(:cancel_account),
+          user_registration_path,
+          data: {:confirm => "Are you sure?"},
```
Comment (sujoydatta) [inline app/views/users/show.html.erb]: i18n

**Comment by MuhammadTamzid** [comment]
Ok

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -3,7 +3,11 @@
 <div class="row">
   <div class="panel panel-default">
     <div class="panel-heading">
-      <div class="pull-right">
+      <div>
+        <% if @user == current_user %>
```
Comment (dieunb) [inline app/views/users/show.html.erb]: use cancancan do implement it

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -13,6 +17,14 @@
       <label><%= t "email" %></label>
       <p class="form-control profile_align"><%= @user.email %></p>
     </div>
-    <div class="panel-footer panel-course-footer"></div>
+    <div class="panel-footer panel-course-footer">
+      <% if @user == current_user %>
```
Comment (dieunb) [inline app/views/users/show.html.erb]: same above

**Comment by MuhammadTamzid** [comment]
Ok

**Pull request merged**